internal/command/command.go

```diff
@@ -0,0 +1,307 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2019-2021 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package command
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/psiemens/sconfig"
+
+	"github.com/onflow/flow-cli/pkg/flow"
+	"github.com/onflow/flow-cli/pkg/flow/gateway"
+	"github.com/onflow/flow-cli/pkg/flow/services"
+	"github.com/onflow/flow-cli/pkg/flow/util"
+	"github.com/onflow/flow-go-sdk/client"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+type RunCommand func(
+	*cobra.Command,
+	[]string,
+	*flow.Project,
+	*services.Services,
+) (Result, error)
+
+type Command struct {
+	Cmd   *cobra.Command
+	Flags interface{}
+	Run   RunCommand
+}
+
+var (
+	FilterFlag      = ""
+	FormatFlag      = ""
+	SaveFlag        = ""
+	RunEmulatorFlag = false
+	HostFlag        = ""
+	LogFlag         = "info"
+	NetworkFlag     = ""
+)
+
+// InitFlags init all the global persistent flags
+func InitFlags(cmd cobra.Command) {
```

Comment:
I just want to make sure that all of these make sense as global flags. Making a note to talk about it

Reply:
They don't and a I will make them private, they were like this because before they were used globaly which was a bad decision anyway